### PR TITLE
Feature/vector ariphmetical operations

### PR DESCRIPTION
**Pull request by alois9866**
Resloves #51.

### CODE REVIEW

File: src/Model/CommonTypes.hs [hunk 1]
```diff
@@ -7,20 +7,36 @@ import Graphics.Gloss.Interface.IO.Game (Key, KeyState(..))
 class UnwrapablePair a where
   unwrap :: a -> (Float, Float)
 
+class LSE a where
```
Comment (dm-gh) [inline src/Model/CommonTypes.hs]: Define LSE

Comment (alois9866) [reply src/Model/CommonTypes.hs]: done

Comment (alois9866) [reply src/Model/CommonTypes.hs]: It is too complicated for the task IMHO

Comment (fizruk) [reply src/Model/CommonTypes.hs]: And `LSE` is too arbitrary and will definitely lead to errors and/or confusion in the code.

**Comment by dm-gh** [comment]
@alois9866 why you closed and then reopened this request?

**Pull request closed without merging**